src/index.js

```diff
@@ -63,14 +71,12 @@ program.action(function(cmd, cmd2) {
       "instead"
     );
   } else {
-    var suggestion = didYouMean(cmd, commandNames);
-    suggestion = suggestion || didYouMean([cmd, cmd2].join(":"), commandNames);
-    if (suggestion) {
-      logger.error();
-      logger.error("Did you mean", clc.bold(suggestion) + "?");
+    if (!suggestCommands(cmd, commandNames)) {
+      cmd = [cmd, cmd2].join(":");
     }
   }
 
+  suggestCommands(cmd, commandNames);
```

Comment:
With `suggestCommands` here, it can print "Did you mean" twice, correct? As mentioned in another comment, this should me moved up to inside the `if (!suggestCommands...)` block above after the `cmd` is combined.